library/Reporting/Mail.php

```diff
@@ -33,7 +33,7 @@ class Mail
      */
     public function getFrom()
     {
-        if (isset($this->from)) {
+        if ($this->from !== null) {
```

Comment:
why is this necessary?

Reply:
@sukhwinder33445 told me already. Please adjust the doctype of this property and $transport instead.

Reply:
No! These properties aren't allowed to be null and never to be used with null values. This check only ensures that this property is not yet set and therefore must be initialized properly.